gui/sessionviewer/src/TProofProgressLog.cxx

```diff
@@ -655,7 +655,10 @@ void TProofProgressLog::Rebuild()
       }
    }
    // Cleanup current TProofLog
-   if (fProofLog) delete fProofLog;
+   if (fProofLog) {
+      delete fProofLog;
+      fProofLog = nullptr;
```

Comment:
As a side note, _historically_ we have used the ROOT macro `SafeDelete` for this pattern.

Reply:
From other side, all kinds of code profilers do not understand them.
Much easier to write three lines and make them (clang-tidy or DeepCode) happy
All my last leaks I found via DeepCode

Reply:
> All my last leaks I found via DeepCode

And thank you for fixing them ! :)

> From other side, all kinds of code profilers do not understand them.

That would be surprising, it literally expand to the same code (so only the code profilers that don't understand C macro will miss it ...)